frontend/src/components/body/hooks/useLocalStorage.ts

```diff
@@ -0,0 +1,23 @@
+import { useCallback } from "react";
+
+const useLocalStorage = (key: unknown) => {
+  const stringKey = JSON.stringify(key);
+
+  const getItemFromLocalStorage = useCallback((): unknown => {
```

Comment:
Should be any and not unknown

Reply:
okay will read the definitions once again